cpp/test/sg/svc_test.cu

```diff
@@ -108,7 +108,7 @@ TYPED_TEST_CASE(WorkingSetTest, FloatTypes);
 TYPED_TEST(WorkingSetTest, Init)
 {
   auto stream = this->handle.get_stream();
-  this->ws    = new WorkingSet<TypeParam>(this->handle, this->handle.get_stream(), 10);
+  this->ws    = new WorkingSet<TypeParam>(this->handle, stream, 10);
```

Comment:
Out of scope for this PR, but @dantegd this fixture is structured a bit oddly... there's really no need for it to be stateful, is there? The Init test could be done without accessing `this` at all (just create a WorkingSet on the stack) and the Select test could initialize everything in the WorkingSetTest constructor by itself since that's the only place that data is used. Am I missing something?

Reply:
yeah, I think you're correct and this could be simplified that way. As a matter of fact, there's a lot of tech debt in our C++ tests that we need to get to in a future refactor of things.